NumLib/TimeStepping/Algorithms/FixedTimeStepping.cpp

```diff
@@ -24,27 +23,33 @@
 
 namespace NumLib
 {
-
-FixedTimeStepping::FixedTimeStepping(double t0, double tn, const std::vector<double> &vec_all_dt)
-: _t_initial(t0), _t_end(computeEnd(t0, tn, vec_all_dt)), _dt_vector(vec_all_dt), _ts_prev(t0), _ts_current(t0)
-{}
+FixedTimeStepping::FixedTimeStepping(double t0,
+                                     double tn,
+                                     const std::vector<double>& vec_all_dt)
+    : ITimeStepAlgorithm(t0,
+                         computeEnd(t0, tn, vec_all_dt),
+                         vec_all_dt,
+                         MathLib::VecNormType::INVALID)
```

Comment:
removed and added a member of
```
isSolutionErrorComputationNeeded()
```
to TimeStepAlgorithm to check whether the error calculation is needed.